README.md

```diff
@@ -1,4 +1,9 @@
-Update 28 November 2020: [Now also available in simplified Chinese](https://github.com/kumakichi/easy_rust_chs) thanks to [kumakichi](https://github.com/kumakichi)!
+## Update
```

Comment:
Circumvents the bug stated in [this issue](https://github.com/Dhghomon/easy_rust/issues/96).